ONSdigital/response-operations-ui
# Store messages survey in redis session

This pull request stores the survey in a redis session meaning that when the user goes back to their messages, it would be on the survey they have chosen instead of going back to the select survey screen.

How to test:

- Check the functionality by selecting a survey and making sure it returns back to the correct messages screen.
- Unit tests run
